src/main/java/dev/enjarai/trickster/datagen/ModBlockTagGenerator.java

```diff
@@ -65,6 +67,14 @@ protected void configure(RegistryWrapper.WrapperLookup wrapperLookup) {
         registerHeatConversion(Blocks.LAPIS_ORE, Blocks.DEEPSLATE_LAPIS_ORE);
         registerHeatConversion(Blocks.DIAMOND_ORE, Blocks.DEEPSLATE_DIAMOND_ORE);
         registerHeatConversion(Blocks.COBWEB, Blocks.AIR);
+        registerHeatConversion(Blocks.COAL_BLOCK, Blocks.FIRE);
```

Comment:
Since this system just takes the default state of a block, I don't think this would work that well...

Reply:
I tested it and it worked without issues

Reply:
Does it also work when the coal is on a wall, with no floor underneath?

Reply:
I would also like to know how it behaves without a floor.

Reply:
> Does it also work when the coal is on a wall, with no floor underneath?

Yes, I tested it on a tree, it used the correct face, likely due to block update from the other fire blocks

Reply:
Those additional fire blocks might not always be created though...

Reply:
fair, #69 fixes this tho :P